debian/rules

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/make -f
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# The following packages explicitly state the version of the linux
+# kernel to be installed, as well as any packages that are kernel
+# specific (e.g. tools like perf, that are tied to the kernel version).
+# Any additional tools that are not specific to the kernel version
+# should be included in the "tools.list.chroot" file.
+#
+
+%:
+	dh $@
+
+override_dh_auto_build:
```

Comment:
Is this needed? We don't have a `Makefile` in this repository, so I don't think this or the `override_dh_auto_test` stuff below is needed, but maybe I'm wrong...